sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -295,12 +337,176 @@ of these enumerators, and it provides a brief description of their meanings.
 |`intel_gpu_12_10_0`
 |1
 |Alias for `intel_gpu_dg1`.
+
+|`nvidia_gpu_sm20`
+|1
```

Comment:
In addition, change all these new entries to `2`, so that users know that these entries were added in version 2 of the specification.

Reply:
thanks @gmlueck. done.